execution_engine/src/core/engine_state/system_contract_cache.rs

```diff
@@ -3,9 +3,8 @@ use std::{
     sync::{Arc, RwLock},
 };
 
-use parity_wasm::elements::Module;
-
 use casper_types::ContractHash;
+use parity_wasm::elements::Module;
```

Comment:
This was correct before - parity_wasm is a 3rd party crate.

Reply:
Regrouped